ui/mirage/factories/evaluation.js

```diff
@@ -31,9 +31,17 @@ const generateCountMap = (keysCount, list) => () => {
 };
```

Comment:
In general, changes to this factory for this PR can be ignored; this is just a prettier / line-length update.